js/actions/windowActions.js

```diff
@@ -255,6 +255,19 @@ const windowActions = {
   },
 
   /**
+   * Dispatches a message to the store to set preview mode.
+   * This will check whether setPreviewFrame should be fired or not.
+   *
+   * @param {Boolean} shouldEnablePreview - true if user enters in previewMode state
+   */
+  setPreviewMode: function (shouldEnablePreview) {
```

Comment:
this action doesn't provide any new information that we don't already have in `setPreviewFrame`. We are already duplicating information in `setTabHoverState` and `setPreviewFrame` (fixed in #9542) and definitely don't want to add any more actions for this functionality